ansible/roles/sunbird-system-init/templates/system_init_commands.j2

```diff
@@ -0,0 +1,323 @@
+                -H 'accept: application/json' \
+                -H 'authorization: Bearer '"$sunbird_api_auth_token" \
+                -d '{
+                        "request": {
+                                "firstName":"'"$sunbird_root_user_firstname"'",
+                                "lastName":"'"$sunbird_root_user_lastname"'",
+                                "userName":"'"$sunbird_root_user_username"'",
+                                "password":"'"$sunbird_root_user_password"'",
+                                "email":"'"$sunbird_root_user_email"'",
+                                "phone":"'"$sunbird_root_user_phone"'",
+                                "channel": "'"$sunbird_custodian_tenant_channel"'",
+                                "emailVerified": true,
+                                "phoneVerified": true
+                        }
+                    }')
+}
+
+addUserToCustodianOrg() {
+        getKeycloakToken
+        add_user_response=$(curl -X POST $proto://$proxy_server_name/api/org/v1/member/add \
+                -H 'Cache-Control: no-cache' \
+                -H 'Content-Type: application/json' \
+                -H 'accept: application/json' \
+                -H 'authorization: Bearer '"$sunbird_api_auth_token" \
+                -H 'x-authenticated-user-token: '"$keycloak_access_token" \
+                -d '{
+                        "request": {
+                                "organisationId": "'"$sunbird_custodian_tenant_id"'",
+                                "userId": "'"$sunbird_root_user_id"'",
+                                "roles": ["ORG_ADMIN"]
+                        }
+                    }')
+}
+
+assignRoleToRootUser() {
+        getKeycloakToken
+        assign_role_response=$(curl -X POST $proto://$proxy_server_name/api/user/v1/role/assign \
+                -H 'Cache-Control: no-cache' \
+                -H 'Content-Type: application/json' \
+                -H 'accept: application/json' \
+                -H 'authorization: Bearer '"$sunbird_api_auth_token" \
+                -H 'x-authenticated-user-token: '"$keycloak_access_token" \
+                -d '{
+                        "request": {
+                                "organisationId": "'"$sunbird_custodian_tenant_id"'",
+                                "userId": "'"$sunbird_root_user_id"'",
+                                "roles": ["ORG_ADMIN"]
+                        }
+                    }')
+}
+
+getValueFromSystemSettings "systemInitialisationStatus";
+if [ -z "$system_settings_value" ];
+then
+        echo "Adding systemInitialisationStatus field to System Settings table...";
+        setValueInSystemSettings "systemInitialisationStatus" "Uninitialised";
+fi
+
+getValueFromSystemSettings "systemInitialisationStatus";
+if [ "$system_settings_value" = "Initialised" ];
+then
+        echo "System already Initialised. Exiting program...";
+        exit 0;
+else
+        echo "Starting System Initialisation...";
```

Comment:
If Cassandra is not up, then system initialisation shouldn't be started. Currently initialisation is being attempted.

Reply:
Review Comment Incorporated in code